packages/api-nest/src/library/library.service.ts

```
@@ -4,122 +4,59 @@ import {
 
 @Injectable()
 export class LibraryService {
   private readonly logger = new Logger(LibraryService.name)
 
   constructor(
-    @InjectRepository(LibraryItemEntity)
-    private readonly libraryRepository: Repository<LibraryItemEntity>,
-    private readonly dataSource: DataSource,
+    @Inject('ILibraryItemRepository')
+    private readonly libraryRepository: ILibraryItemRepository,
     private readonly eventBus: EventBusService,
```

ESLint

[error] Delete `,`

(prettier/prettier)